networkx/algorithms/community/community.py

```diff
@@ -0,0 +1,79 @@
+#-*- coding: utf-8 -*-
+import networkx as nx
+
+__all__ = ['girvan_newman']
+
+
+def girvan_newman(G, weight=None):
+    """Find communities in graph using Girvan-Newman method.
```

Comment:
Use an en dash here instead of a hyphen (``Girvan–Newman`` instead of ``Girvan-Newman``). This applies elsewhere as well.